scripts/docker_run

```diff
@@ -13,15 +16,22 @@ docker build \
   --build-arg="USERNAME=$USERNAME" \
   --build-arg="UID=$(id -u)" \
   --build-arg="GID=$(id -g)" \
-  .
+  --quiet \
+  . > /dev/null
+
+docker_run_flags=(
+  "--interactive"
+  "--tty"
+  "--volume=$PWD/bazel-cache:/home/$USERNAME/.cache/bazel"
+  "--volume=$PWD/cargo-cache:/home/$USERNAME/.cargo/registry"
+  "--volume=$PWD:/opt/my-project"
```

Comment:
Nit: Can we match the format for the other two volumes? (is `:` needed?)

Reply:
The `:` is needed to map host source directory to guest destination.

`-v, --volume=[host-src:]container-dest[:<options>]:`
From https://docs.docker.com/engine/reference/run/#volume-shared-filesystems

Reply:
Doh! Thanks for explaining....